crosswalks/GitLab.csv

```diff
@@ -0,0 +1,74 @@
+Property,GitLab
+codeRepository,web_url
+programmingLanguage,/projects/:id/languages
+review,
+runtimePlatform,
+targetProduct,
+applicationCategory,
+applicationSubCategory,
+downloadUrl,http_url_to_repo
```

Comment:
```suggestion
downloadUrl,
```

https://schema.org/downloadUrl says "If the file can be downloaded, URL to download the binary." which does not match `http_url_to_repo`.